.github/workflows/reusable/ci-smoke-eval.yml

```
@@ -0,0 +1,54 @@
+        type: string
+        default: '[]'
+      expected-version:
+        description: Version string the deployment should report.
+        type: string
+        default: ''
+    secrets:
+      anthropic-api-key:
+        description: Anthropic API key.
+        required: true
+
+outputs:
```

actionlint (1.7.12)

[error] unexpected key "outputs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)